### PR TITLE
Use ST formatting for console printing, and clean up some unnecessary sprintf

### PR DESCRIPTION
**Pull request by zrax**
This is the first step towards cleaning up the console with string_theory

### CODE REVIEW

File: Sources/Plasma/FeatureLib/pfConsole/pfCCRConsoleCommands.cpp [hunk 1]
```diff
@@ -62,8 +62,6 @@ You can contact Cyan Worlds, Inc. by email legal@cyan.com
 PF_CONSOLE_FILE_DUMMY(CCR)
 //// DO NOT REMOVE!!!!
```
Comment (Hoikas) [inline Sources/Plasma/FeatureLib/pfConsole/pfCCRConsoleCommands.cpp]: *finger hovers over the delete key*

File: Sources/Plasma/FeatureLib/pfConsoleCore/pfConsoleCmd.h [hunk 2]
```diff
@@ -282,4 +284,10 @@ class pfConsoleCmdIterator
 #define PF_CONSOLE_FILE_DUMMY( name ) \
     void _console_##name##_file_dummy() { }
 
+template <typename... Args>
+void pfConsolePrintF(void pfun(const char *), const char *fmt, Args &&...args)
```
Comment (Hoikas) [inline Sources/Plasma/FeatureLib/pfConsoleCore/pfConsoleCmd.h]: Style nit (pointer binds to the type, not the argument)

Comment (zrax) [reply Sources/Plasma/FeatureLib/pfConsoleCore/pfConsoleCmd.h]: I tend to follow the prevailing style in the modified file for pointer/reference alignment...  In this case, the vast majority of the file binds to the name, not the type.

Comment (Hoikas) [reply Sources/Plasma/FeatureLib/pfConsoleCore/pfConsoleCmd.h]: I guess we can save the formatting fixes for later in that case.

**Pull request merged**